app/models/beard.rb

```diff
@@ -0,0 +1,35 @@
+class Beard < ActiveRecord::Base
+  belongs_to :user
+
+  STATUS = ['shaved', 'trimmed', 'unset']
+
+  validates :status,  presence: true
+  validates :user_id, presence: true
+
+  validate :valid_status
+
+  def update_status!(params)
+    self.status = params[:status] if params[:status]
+    self.save
+
+    self.toggle_massaged! if params[:is_massaged]
```

Comment:
Seems like this method should be private?  Also, you don't need to use "self." for methods within the object unless there is an overscoping variable.